src/cms/templates/organizations/organization_list_row.html

```diff
@@ -11,9 +11,9 @@
 	</td>
 	<td class="pr-2">
         {% if organization.icon %}
-            <a href="{{ organization.icon.url }}" target="_blank" rel="noopener noreferrer" class="bg-transparent hover:underline text-blue-500 hover:text-blue-600 cursor-pointer">
-                {{ organization.icon.url }}
-            </a>
+        <a href="{% url 'edit_organization' organization_id=organization.id %}">
+            <img src="{{organization.icon.url}}" alt="{{organization.icon.url}}" style="height:60px">
```

Comment:
```suggestion
            <img src="{{organization.icon.url}}" alt="{{organization.icon.url}}" style="max-height:48px">
```
I'd go with this max height, because the normal text cells also have this height (24px for the content and 12px for top and bottom padding)... this way, all rows are of equal height, no matter whether an icon is set or not.